docs/02_tokenization_files/figure-html/tokendists-1.svg

```diff
@@ -25,20 +25,20 @@
   </clipPath>
 </defs>
 <g clip-path='url(#cpMTI5LjI1fDQ5OC41MnwzNy41NHwyNzkuNzI=)'>
-<polyline points='205.35,279.72 205.35,37.54 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
+<polyline points='205.34,279.72 205.34,37.54 ' style='stroke-width: 0.53; stroke: #EBEBEB; stroke-linecap: butt;' />
```

Comment:
Same as above: switch to using a seed for jitter

Reply:
Done!